src/main/kotlin/step3/controller/RacingController.kt

```diff
@@ -0,0 +1,52 @@
+package step3.controller
+
+import step3.model.RacingProcess
+import step3.model.RacingProcessFactory
+import step3.model.RandomGenerator
+import step3.model.RandomGeneratorFactory
+import step3.view.InputView
+import step3.view.InputViewFactory
+import step3.view.ResultView
+import step3.view.ResultViewFactroy
+
+fun main() {
+    val inputView: InputView = InputViewFactory.newInstance()
```

Comment:
kotlin은 타입추론이 가능하기때문에 타입을 명시하지않아도괜찮은데요!
모든 변수에 타입을 명시하신 이유가 있을까요?

```suggestion
    val inputView = InputViewFactory.newInstance()
```